src/graphql/fragments/expenditures.graphql

```diff
@@ -52,6 +52,7 @@ fragment Expenditure on Expenditure {
     filter: {
       or: [
         { type: { eq: MOVE_FUNDS } }
+        { type: { eq: MOVE_FUNDS_MOTION } }
```

Comment:
What was the reason behind adding this? Did you come across some funding motions misclassified as move funds?